src/pages/PrintPage.tsx

```diff
@@ -190,8 +198,8 @@ class SummaryPage extends React.Component<RouteComponentProps, State> {
               <Content>
                 <BallotSelections>
                   <BallotContext.Consumer>
-                    {({ election, votes }) =>
-                      election!.contests.map(contest => (
+                    {({ votes }) =>
+                      contests.map(contest => (
```

Comment:
Would probably be easier to follow code if context was accessed in one place, rather than two.